tests/FunctionsTest.php

```diff
@@ -152,14 +144,6 @@ public function provideReadConcernsAndDocuments()
         ];
     }
 
-    /**
-     * @dataProvider provideWriteConcernsAndDocuments
-     */
-    public function testWriteConcernAsDocument(WriteConcern $writeConcern, $expectedDocument)
-    {
-        $this->assertEquals($expectedDocument, \MongoDB\write_concern_as_document($writeConcern));
-    }
-
     public function provideWriteConcernsAndDocuments()
```

Comment:
The data providers should be suitable to remove as well:

 * `provideReadConcernsAndDocuments()`
 * `provideWriteConcernsAndDocuments()`